### ISSUE TITLE
MKSYSVINIT make fails

### ISSUE THREAD

**Issue by udeved** [OpenRC/openrc#141]
The make with these args fails with 0.27.1

```
_args=(SYSCONFDIR=/etc)
_args+=(PREFIX=/usr)
_args+=(SBINDIR=/usr/bin)
_args+=(LIBEXECDIR=/usr/lib/rc)
_args+=(MKSELINUX=no)
_args+=(MKPAM=pam)
_args+=(MKTERMCAP=ncurses)
_args+=(MKNET=no)
_args+=(MKSYSVINIT=yes)

```

The error thrown.

```
make[1]: Leaving directory '/build/openrc/src/openrc-0.27.1/man'
===> scripts (install)
make[1]: Entering directory '/build/openrc/src/openrc-0.27.1/scripts'
install -d /build/openrc/pkg/openrc/usr/bin
install -d /build/openrc/pkg/openrc//usr/lib/rc/bin
ln -sf ..//usr/lib/rc/bin/rc-sstat /build/openrc/pkg/openrc//usr/bin/rc-sstat
ln -sf  ..//usr/lib/rc/bin/halt /build/openrc/pkg/openrc/sbin/halt
install -m 0755 on_ac_power rc-sstat halt poweroff reboot shutdown /build/openrc/pkg/openrc//usr/lib/rc/bin
ln: failed to create symbolic link '/build/openrc/pkg/openrc/sbin/halt': No such file or directory
make[1]: *** [Makefile:21: _installafter] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/build/openrc/src/openrc-0.27.1/scripts'
make: *** [/build/openrc/src/openrc-0.27.1//mk/subdir.mk:31: realinstall] Error 2

```

I perhaps find time to create a patch, it looks like the scripts/Makefile needs slight enhancement.

**Comment by udeved** [comment]
I think the PR fixes the build issues ad the system is fine now.
https://github.com/OpenRC/openrc/pull/142

Is there any upgrade advise to make the transition from sysvinit to openrc-init smooth?

Example:
The system is openrc-0.26.3 and sysvinit.
If I do the upgrade to 0.27.1 which removes sysvinit, I am stuck at console trying to reboot with reexec the init, to no effect somehow.

**Comment by udeved** [comment]
Reason to make it absolute path was this...

`ln -sf ..//usr/lib/rc/bin/rc-sstat /build/openrc/pkg/openrc//usr/bin/rc-sstat`

The double slash renders the symlink broke.

**Comment by udeved** [comment]
@williamh 

I think the issue is not solved yet.
You got this in master now

```
ln -sf ${DIR}/reboot ${DESTDIR}/${SBINDIR}/reboot
ln -sf	 openrc-shutdown ${DESTDIR}/${SBINDIR}/shutdown
ln -sf openrc-init ${DESTDIR}/${SBINDIR}/init
```

However, shouldn't there be a symlink for the ${DIR}/shutdown too? Probably replace the direct link to openrc-shutdown, since ${DIR}/shutdown script calls openrc-shutdown with args like in original state before the two patches?

```
ln -sf ${DIR}/reboot ${DESTDIR}/${SBINDIR}/reboot
ln -sf	 ${DIR}/shutdown ${DESTDIR}/${SBINDIR}/shutdown
ln -sf openrc-init ${DESTDIR}/${SBINDIR}/init
```

**Comment by williamh** [comment]
This is fixed in 2f81c100.
